cuegui/cuegui/MenuActions.py

```diff
@@ -637,7 +637,7 @@ def retryDead(self, rpcObjects=None):
             if cuegui.Utils.questionBoxYesNo(self._caller, "Confirm",
                                              "Retry all DEAD frames in selected layers?",
                                              [layer.data.name for layer in layers]):
-                layers[-1].parent.retryFrames(layer=[layer.data.name for layer in layers],
+                layers[-1].parent().retryFrames(layer=[layer.data.name for layer in layers],
                                               state=[opencue.api.job_pb2.DEAD])
```

Comment:
nit: Bump this over to the right so it's aligned with the `layer=` arg above it.